### PR TITLE
add accessors module with st_envelope

### PR DESCRIPTION
**Pull request by vdelacruzb**

### CODE REVIEW

File: accessors/bq/doc/REFERENCE.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+## accessors
+
+<div class="badge core"></div>
+
+TODO: module description.
```
Comment (alex-graciano) [inline accessors/bq/doc/REFERENCE.md:5]: Missing description and documentation of `ST_ENVELOPE`

**Review by alex-graciano — CHANGES_REQUESTED** [review]
Nicely done! But I miss the documentation of both the module and the function

File: accessors/bq/doc/REFERENCE.md (hunk 1 above)
Comment (vdelacruzb) [reply accessors/bq/doc/REFERENCE.md:5]: True missed, totally ^^'. Going for it

Comment (alex-graciano) [reply accessors/bq/doc/REFERENCE.md]: Cool!

**Review by alex-graciano — APPROVED** [review]
LGTM!

**Comment by Jesus89** [comment]
Hey, the date is not correct `2021-05-03` -> `2021-05-04`. But we can fix it in the next PR

**Pull request merged**